src/components/NewDesign/Inputs/ExchangeAmountInput/ExchangeAmountInput.tsx

```diff
@@ -0,0 +1,103 @@
+import { Input } from '@material-ui/core'
+import React, { CSSProperties, useRef } from 'react'
+import classNames from 'classnames'
+import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
+import Select from '@components/NewDesign/Inputs/Select/Select'
+import useStyles from './style'
+
+interface IProps {
+  setValue: (value: string) => void
+  value?: string
+  error?: string | null
+  className?: string
+  placeholder?: string
+  style?: CSSProperties,
+  onMaxClick: () => void,
+  current: string | null
+  tokens: Array<{ symbol: string, name: string, logoURI: string }> | null
```

Comment:
remove "| null" everywhere where tokens list is used

Reply:
done